### PR TITLE
Schema changes for HH accomodation type Eng & Wal

### PR DESCRIPTION
**Pull request by JamesGardiner**
### What is the context of this PR?
Schema changes for Household England and Wales Survey accommodation section.

### How to review 
Build the new schema changes. Test that the house hold accommodation flow is as specified in  https://trello.com/c/s3FsJSyh/3209-household-accommodation-eng-wls

### CODE REVIEW

**Comment by berroar** [comment]
[Household accommodation Eng/Wls](https://trello.com/c/s3FsJSyh/3209-household-accommodation-eng-wls)

**Review by berroar — COMMENTED** [review]
The default routing looks good and I am able to skip through the non mandatory questions

File: data-source/jsonnet/england-wales/lib/rules.libsonnet [hunk 1]
```diff
@@ -98,4 +98,8 @@
     condition: 'equals',
     value: 'Flat, maisonette or apartment',
   },
+  accomodationNotAnswered: {
```
Comment (MebinAbraham) [inline data-source/jsonnet/england-wales/lib/rules.libsonnet]: This doesn't seem to be used.

**Review by MebinAbraham — CHANGES_REQUESTED** [review]
Routing/Validation is fine.

Translation POT files need to be updated.

Also:

**Review by MebinAbraham — APPROVED** [review]

**Review by berroar — APPROVED** [review]

**Pull request merged**